index.html

```diff
@@ -200,7 +200,7 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 <h3>
 <a id="same-example-but-this-time-the-data-is-supplied-as-a-command-line-argument" class="anchor" href="#same-example-but-this-time-the-data-is-supplied-as-a-command-line-argument" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Same example, but this time the data is supplied as a command line argument:</h3>
 
-<pre><code>$ java -jar generate.war -t eiffelactivityfinished -json {"msgParams": {"meta": {"type":"EiffelActivityFinishedEvent","domainId": "example.domain","tags": ["tag1", "tag2"],"source": {"host": "host","name": "name","uri": "http://java.sun.com/j2se/1.3/","serializer": {"groupId": "G","artifactId": "A",       "version": "V"}}}},"eventParams": {"data": {"outcome": {"conclusion": "TIMED_OUT","description": "Compilation timed out."},"persistentLogs": [{"name": "firstLog","uri": "http://myHost.com/firstLog"},{"name": "otherLog","uri": "isbn:0-486-27557-4"}]},"links": {"activityExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1","causes": ["cause1", "cause2"],"flowContext": "flowContext"}}}
+<pre><code>$ java -jar generate.war -t EiffelActivityFinishedEvent -json {"msgParams": {"meta": {"type":"EiffelActivityFinishedEvent","domainId": "example.domain","tags": ["tag1", "tag2"],"source": {"host": "host","name": "name","uri": "http://java.sun.com/j2se/1.3/","serializer": {"groupId": "G","artifactId": "A",       "version": "V"}}}},"eventParams": {"data": {"outcome": {"conclusion": "TIMED_OUT","description": "Compilation timed out."},"persistentLogs": [{"name": "firstLog","uri": "http://myHost.com/firstLog"},{"name": "otherLog","uri": "isbn:0-486-27557-4"}]},"links": {"activityExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1","causes": ["cause1", "cause2"],"flowContext": "flowContext"}}}
```

Comment:
White space between artifactId and Version shall be removed.
{"groupId": "G","artifactId": "A",       "version": "V"}}}}